buildconfig/pygame-stubs/image.pyi

```diff
@@ -1,24 +1,18 @@
-from typing import Optional, Tuple, List, Union, IO, Literal
+from typing import IO, Literal, Optional, Sequence, Tuple, Union
 
-from pygame.surface import Surface
 from pygame.bufferproxy import BufferProxy
+from pygame.surface import Surface
 
 _BufferStyle = Union[BufferProxy, bytes, bytearray, memoryview]
-_to_string_format = Literal['p', 'RGB', 'RGBX', 'RGBA', 'ARGB', 'RGBA_PREMULT', 'ARGB_PREMULT']
-_from_buffer_format = Literal['p', 'RGB', 'BRG', 'RGBX', 'RGBA', 'ARGB']
-_from_string_format = Literal['p', 'RGB', 'RGBX', 'RGBA', 'ARGB']
+_to_string_format = Literal["p", "RGB", "RGBX", "RGBA", "ARGB", "RGBA_PREMULT", "ARGB_PREMULT"]
+_from_buffer_format = Literal["p", "RGB", "BRG", "RGBX", "RGBA", "ARGB"]
+_from_string_format = Literal["p", "RGB", "RGBX", "RGBA", "ARGB"]
 
-def load(filename: Union[str, IO], namehint: Optional[str] = "") -> Surface: ...
-def save(surface: Surface, filename: Union[str, IO]) -> None: ...
+def load(filename: Union[str, IO[bytes]], namehint: Optional[str] = ...) -> Surface: ...
```

Comment:
What is this change for with filename? From`IO` -> `IO[bytes]`

Reply:
The IO can be bytes or str, in this case I figured it had to be bytes